developer-workflow/c-api.rst

```diff
@@ -53,14 +60,18 @@ Private names
 --------------
 
 Any API named with a leading underscore is also considered internal.
-There are two main use cases for using such names rather than putting the
-definition in ``Include/internal/`` (or directly in a ``.c`` file):
+There is currently only one main use case for using such names rather than
+putting the definition in ``Include/internal/`` (or directly in a ``.c`` file):
+
+* Internal helpers for other public API, which users should not call directly.
+
+Note that historically, underscores were used for API that is better served by
```

Comment:
```suggestion
Note that historically, underscores were used for APIs that are better served by
```

Fix plural spelling and singular-plural agreement problem (which initially confused me as to whether you meant it as plural at all)

Reply:
An “application programming interface” is (roughly) a *collection* of functions, types, macros, etc. Ideally there'd be a good general word for one of those.
*API* is the best I know, but it's overloading the term in a confusing way.

Reply:
Yeah; the real linguistic issue is there's no real noun in English that spans for all the _elements_ (functions, types, macros, etc) that together fall under an API. It's not ideal, but its standard convention and the best we have.